content/posts/newsletter-008/index.md

```diff
@@ -51,6 +51,18 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [DynaMaze][dynamaze]
+
+![DynaMaze promotional image](dynamaze.png)
+
+[DynaMaze][dynamaze] is an [open source][dynamaze-source] multiplayer
```

Comment:
maybe add a few words about what kind of gameplay the game is about? Something like how it's described on the itch page

> DynaMaze is a multiplayer strategy game about traversing a shifting maze. Adjust the maze to build a path to your target and keep the other players from getting to theirs.